src/components/Reports/GoalCalculator/CalculatorSettings/Steps/InformationStep/InformationStepForm/InformationStepPersonalForm.tsx

```diff
@@ -130,7 +130,7 @@ export const InformationStepPersonalForm: React.FC<
                   {...field}
                   fullWidth
                   size="small"
-                  label={t('Location')}
+                  label={t('Ministry Location')}
```

Comment:
I know this isn't part of this PR, but it looks like the children inpue only shows when spouse is null | undefined | false. Is this correct? As I would assume a few people will have a spouse and children.

Reply:
`IsSpouse` is true when rendering the spouse's personal information form. The idea is that the `children` input shows in the primary person's form and not in the spouse's form. The assumption is that the person wouldn't have a separate list of children from their spouse.